app/src/main/java/com/spoony/spoony/presentation/placeDetail/component/StoreInfoItem.kt

```diff
@@ -0,0 +1,159 @@
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.vector.ImageVector
+import androidx.compose.ui.res.vectorResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.style.TextOverflow
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import com.spoony.spoony.R
+import com.spoony.spoony.core.designsystem.theme.SpoonyAndroidTheme
+
+@Composable
+fun StoreInfoItem(
+    title: String,
+    subTitle: String? = null,
+    items: List<String>,
+    isMenu: Boolean = true,
+    modifier: Modifier = Modifier
+) {
+    val (shape, paddingValues) = remember(isMenu) {
+        when (isMenu) {
+            true -> RoundedCornerShape(
+                topStart = 8.dp,
+                topEnd = 8.dp,
+                bottomStart = 20.dp,
+                bottomEnd = 20.dp
+            ) to PaddingValues(
+                top = 20.dp,
+                bottom = 28.dp,
+                start = 16.dp,
+                end = 16.dp
+            )
+            false -> RoundedCornerShape(
+                topStart = 20.dp,
+                topEnd = 20.dp,
+                bottomStart = 8.dp,
+                bottomEnd = 8.dp
+            ) to PaddingValues(
+                vertical = 22.dp,
+                horizontal = 16.dp
+            )
+        }
+    }
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .background(
+                shape = shape,
+                color = SpoonyAndroidTheme.colors.gray0
+            )
+            .padding(paddingValues)
+    ) {
+        Text(
+            title,
+            style = SpoonyAndroidTheme.typography.body1b
+        )
+        Spacer(modifier = Modifier.height(12.dp))
+        if (subTitle != null) {
+            Text(
+                subTitle,
+                style = SpoonyAndroidTheme.typography.title2sb
+            )
+            Spacer(modifier = Modifier.height(11.dp))
+        }
+        Column(
+            verticalArrangement = Arrangement.spacedBy(12.dp)
+        ) {
+            items.forEach { item ->
+                IconText(
+                    icon = if (isMenu) R.drawable.ic_spoon_24 else R.drawable.ic_pin_24,
+                    iconSize = 20,
+                    text = item,
+                    textStyle = SpoonyAndroidTheme.typography.body2m,
+                    modifier = Modifier.fillMaxWidth()
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun IconText(
+    icon: Int,
```

Comment:
```suggestion
    @DrawableRes icon: Int,
```
P3: 필수는 아니지만,, 이렇게 Drawable id만 받을 수 있도록 설정해주는 것도 좋을 것 같아요!

Reply:
ImageVector로 수정했습니다!